### ISSUE TITLE
Issues during installation on Rails 5

### ISSUE THREAD

**Issue by chrisedington** [joshmn/caffeinate#5]
Hi @joshmn,

me again - migrations were fixed thanks!

I'm having some more issues -- put the basics in place from the website homepage and getting the following:
```
➜  xx git:(ce/drip-campaign) ✗ ruby -v
ruby 2.4.9p362 (2019-10-02 revision 67824) [x86_64-darwin19]
➜  xx git:(ce/drip-campaign) ✗ rails -v
Rails 5.0.7.2
➜  xx git:(ce/drip-campaign) ✗
```
```
➜  xx git:(ce/drip-campaign) ✗ rails s
/Users/chris/.rvm/gems/ruby-2.4.9/gems/caffeinate-0.15.0/lib/caffeinate/schedule_evaluator.rb:23:in `<class:ScheduleEvaluator>': undefined method `delegate_missing_to' for Caffeinate::ScheduleEvaluator:Class (NoMethodError)
Did you mean?  DelegateClass
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/caffeinate-0.15.0/lib/caffeinate/schedule_evaluator.rb:22:in `<module:Caffeinate>'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/caffeinate-0.15.0/lib/caffeinate/schedule_evaluator.rb:1:in `<main>'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:293:in `block in require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:259:in `load_dependency'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:293:in `require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/caffeinate-0.15.0/lib/caffeinate/drip.rb:4:in `<main>'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:293:in `block in require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:259:in `load_dependency'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:293:in `require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/caffeinate-0.15.0/lib/caffeinate.rb:16:in `<main>'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
	from /Users/chris/.rvm/rubies/ruby-2.4.9/lib/ruby/site_ruby/2.4.0/bundler/runtime.rb:81:in `block (2 levels) in require'
	from /Users/chris/.rvm/rubies/ruby-2.4.9/lib/ruby/site_ruby/2.4.0/bundler/runtime.rb:76:in `each'
	from /Users/chris/.rvm/rubies/ruby-2.4.9/lib/ruby/site_ruby/2.4.0/bundler/runtime.rb:76:in `block in require'
	from /Users/chris/.rvm/rubies/ruby-2.4.9/lib/ruby/site_ruby/2.4.0/bundler/runtime.rb:65:in `each'
	from /Users/chris/.rvm/rubies/ruby-2.4.9/lib/ruby/site_ruby/2.4.0/bundler/runtime.rb:65:in `require'
	from /Users/chris/.rvm/rubies/ruby-2.4.9/lib/ruby/site_ruby/2.4.0/bundler.rb:114:in `require'
	from /Users/chris/Repositories/shopstar/config/application.rb:8:in `<main>'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/railties-5.0.7.2/lib/rails/commands/commands_tasks.rb:88:in `block in server'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/railties-5.0.7.2/lib/rails/commands/commands_tasks.rb:85:in `tap'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/railties-5.0.7.2/lib/rails/commands/commands_tasks.rb:85:in `server'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/railties-5.0.7.2/lib/rails/commands/commands_tasks.rb:49:in `run_command!'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/railties-5.0.7.2/lib/rails/commands.rb:18:in `<main>'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/bootsnap-1.7.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
	from /Users/chris/Repositories/shopstar/bin/rails:11:in `<top (required)>'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/spring-2.0.2/lib/spring/client/rails.rb:28:in `load'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/spring-2.0.2/lib/spring/client/rails.rb:28:in `call'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/spring-2.0.2/lib/spring/client/command.rb:7:in `call'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/spring-2.0.2/lib/spring/client.rb:30:in `run'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/spring-2.0.2/bin/spring:49:in `<top (required)>'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/spring-2.0.2/lib/spring/binstub.rb:31:in `load'
	from /Users/chris/.rvm/gems/ruby-2.4.9/gems/spring-2.0.2/lib/spring/binstub.rb:31:in `<top (required)>'
	from /Users/chris/.rvm/rubies/ruby-2.4.9/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:65:in `require'
	from /Users/chris/.rvm/rubies/ruby-2.4.9/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:65:in `require'
	from /Users/chris/Repositories/shopstar/bin/spring:14:in `<top (required)>'
	from bin/rails:3:in `load'
	from bin/rails:3:in `<main>'
```

Seems like `delegate_missing_to` is only implemented in Rails 5.1. Any way to make this backwards compatible?

**Comment by joshmn** [comment]
hey Chris! 

Sorry I'm late. Pushing a fix meow.

**Comment by joshmn** [comment]
fixed in #6 and v0.16.0 :)
